frontend/src/Components/Pages/Explore.jsx

```diff
@@ -1,11 +1,44 @@
-import React from 'react';
-
-function Explore() {
-    return (
-        <div>
-            <h1>Explore Page</h1>
-        </div>
-    );
-}
+import React from "react";
+import axios from "axios";
+
+class Explore extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            data:[]
+        }
+    }
+
+    async componentDidMount() {
+        try {
+            let blogs = await axios.get(`http://localhost:3100/blog/all`);
```

Comment:
Proxy was set up in the frontend. No lone need to add `http://localhost:3100` any more. Please remove that and leave only `/blog/all`